client/.env.docker

```diff
@@ -0,0 +1,4 @@
+PORT=1729
+REACT_APP_HTTP_BACKEND_URL=http://pyrog-server:4000
+REACT_APP_WS_BACKEND_URL=ws://pyrog-server:4000
+REACT_APP_CLEANING_SCRIPTS_URI=http://cleaning-scripts:5001
```

Comment:
can you add a cleaning-script service in the docker-compose.yml now that you setup it ?

Reply:
Yeah I will!